Makefile

```diff
@@ -301,18 +301,6 @@ VIRTENV='virtualenv==20.21.0'
 # https://stackoverflow.com/questions/63515454/why-does-pip3-install-pipenv-give-error-error-cannot-uninstall-distlib
 install-deps-ALPINE-for-pysemgrep:
 	apk add --no-cache $(ALPINE_APK_DEPS_PYSEMGREP)
-# This uninstall step is here because of some weirdness with how
-# `pip install virtualenv==20.21.0` interacts with the version of `virtualenv`
-# on the current Docker image (which is 20.24.5). We get the same attribute error
-# as is described above.
-# The previous Docker image (with virtualenv 20.23.0) has no issues with the
-# `pip install` step following this, but for some reason the newer image does.
-# This version of `virtualenv` was installed along with a newer version of
-# `pre-commit`, and seems to be one of the only disparities which is causing this
-# issue to recur.
-# The easiest solution was to just cause an uninstall prior to the reinstallation
-# step, as it will allow the test to pass.
-	pip uninstall -y virtualenv
```

Comment:
Should we keep this @tpetr  ?

Reply:
This is safe to remove. It was inserted to fix a build issue induced by our updated version of `virtualenv` upon upgrading our `ocaml-layer` image.

This will probably need to be reinserted whenever we end upgrading to 5.1, though.